src/frontend/src/optimizer/property/order.rs

```diff
@@ -55,7 +55,7 @@ impl fmt::Display for Order {
     }
 }
 
-#[derive(Clone)]
+#[derive(Clone, PartialEq)]
 pub struct FieldOrder {
```

Comment:
Can we reuse `common::OrderPair` in the frontend? @st1page

Reply:
TODO 🥵